src/java.base/share/classes/java/lang/invoke/VarHandles.java

```diff
@@ -629,6 +628,8 @@ private static void noCheckedExceptions(MethodHandle handle) {
                     noCheckedExceptions((MethodHandle) arg);
                 }
             }
+        } else {
+            /* TODO: Add code to check for checked exceptions in OpenJ9 MethodHandles. */
```

Comment:
Still need to figure-out ... how to check if OpenJ9 Methodhandles throw checked exceptions?